deepchem/trans/transformers.py

```diff
@@ -2469,3 +2469,120 @@ def median_filter(self, size):
     image = Image.fromarray(self.Image)
     image = image.filter(ImageFilter.MedianFilter(size=size))
     return np.array(image)
+
+
+class RxnSplitTransformer(Transformer):
+  """Splits the reaction SMILES input into the source and target strings
+  required for machine translation tasks.
+
+  The input is expected to be in the form reactant>reagent>product. The source
+  string would be reactants>reagents and the target string would be the products.
+
+  The transformer can also separate the reagents from the reactants for a mixed
+  training mode. This can be toggled (default True) by setting the value of
+  sep_reagent while calling the transformer.
+
+  Examples
+  --------
+  >>> # When mixed training is toggled.
```

Comment:
What is mixed training? Can you add more text above that explains the concept?

Reply:
It just refers to the fact that the reagents and reactants are no longer separated by a '>'. I will clarify this on the doc as well!